### DISCUSSION TITLE
Support for components in dropdown

### DISCUSSION THREAD

**Discussion by KingPinged** [heroui-inc/heroui#597] [Feature requests]
Currently, it is too difficult to create components such as switches inside the dropdown component. If switches were added as an option for dropdown items, it would greatly enhance Next UI and my development because it gives the options that extend upon dropdown currently being used as a form selector to be used in things like settings toggles and dropdowns inside dropdowns. 

I am willing to contribute to this feature

**Comment by jrgarciadev** [comment]
Hey @KingPinged thanks for the suggestion, we will look into this subject 👍🏻
